zanata-war/src/main/java/org/zanata/rest/service/DateRange.java

```diff
@@ -0,0 +1,66 @@
+package org.zanata.rest.service;
+
+import java.util.Date;
+import java.util.TimeZone;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.Days;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.zanata.exception.InvalidDateParamException;
+import org.zanata.util.DateUtil;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+* @author Patrick Huang
+*         <a href="mailto:pahuang@redhat.com">pahuang@redhat.com</a>
+*/
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+class DateRange {
```

Comment:
Nice :+1: 
I wonder if there's a way to make Resteasy do the mapping automatically for us.
Maybe in V 3.0